spec/features/controllers/park_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.configure do |config|
+  config.include Devise::Test::ControllerHelpers, type: :controller
+end
+
+RSpec.describe Api::V1::ParksController, type: :controller do
+
+    describe "GET #show" do
+
+    let!(:state) { FactoryBot.create(:state) }
+    let!(:park) { FactoryBot.create(:park) }
+
+    context "when user goes to park show page" do
+      before {get :show, params: { id: 42 , state_id: 1} }
+      
+      it "returns a specific park" do
+        get :show, params: { id: 42 , state_id: 1}
```

Comment:
Because you have this in a before, you shouldn't need lines 18 or 23

Reply:
Deleted these lines